ProjectSettings/ProjectVersion.txt

```diff
@@ -1,2 +1,2 @@
-m_EditorVersion: 2021.3.2f1
-m_EditorVersionWithRevision: 2021.3.2f1 (79cbdb13f624)
+m_EditorVersion: 2021.3.9f1
```

Comment:
Make sure to commit only the changes related to your PR
With this, you're bumping the editor version for everyone. With that editor bump comes Lobby updates which our code base isn't ready to accept yet (LP and Fernando are working on this).

Reply:
same with manifest.json and packages-lock.json